tests/tracking/test_rest_tracking.py

```diff
@@ -2565,49 +2565,45 @@ def _exclude_system_tags(tags: dict[str, str]):
     }
 
 
-def test_start_trace(mlflow_client):
-    experiment_id = mlflow_client.create_experiment("start end trace")
-
-    # Trace CRUD APIs are not directly exposed as public API of MlflowClient,
-    # so we use the underlying tracking client to test them.
-    client = mlflow_client._tracing_client
+@mock.patch("mlflow.tracing.export.mlflow_v3._logger")
```

Comment:
what calls `mlflow.tracing.export.mlflow_v3._logger`? can we use context manager?